### ISSUE TITLE
Fine-tune SqueezeDet from sparse labels

### ISSUE THREAD

**Issue by cygerts** [BichenWuUCB/squeezeDet#117]
Hi,
I am trying to do domain adaptation with SqueezeDet having sparse labels. So for example there might be many cars  in the image, but only few are annotated. Feeding such image with sparse annotations will of course confuse SqueezeDet.
I have one ideat for doing that:
- modify loss function. So basically we only want to penalize network ONLY for not detecting the bounding box. All false positives should be ignored. How can I achieve that? I suspect that zeroizing LOSS_COEF_CONF_POS or LOSS_COEF_CONF_NEG might do the trick. Also I am not really sure what is the difference between those variables, it is not yet clear for me from the code or article.

I suspect there might be another way of doing that but that's probably the easiest. Any suggestion / comment here would be very helpful.
